### DISCUSSION TITLE
お気に入りの Gem

### DISCUSSION THREAD

**Discussion by hatappi** [roppongirb/discussions#26] [Ideas]

**Comment by tabuchik** [comment]
よく使う
https://github.com/roidrage/lograge

気になってる
https://github.com/github/view_component

**Reply by okuramasafumi** [reply]
https://kaigionrails.org/2021/talks/yykamei/
Logrageに関する発表がKaigi on Railsでありました、それからだいぶ更新入っていますね！

**Comment by scrpgil** [comment]
お世話になってるgem
名前：newrelic_rpm
https://github.com/newrelic/newrelic-ruby-agent

入れて良かったgem
名前：oj
種類：JSONシリアライザ
効果：20%くらい速くなった
https://github.com/ohler55/oj

検討中のgem
名前：sidekiq-scheduler
https://github.com/moove-it/sidekiq-scheduler

**Reply by okuramasafumi** [reply]
ojは`to_json`メソッドをC拡張の実装に上書きしているのでインストールするだけで速くなるんですよね、すごい。

**Comment by hatappi** [comment]
Roppongi.rb の活動報告をこれで作ろうと思っている
https://jekyllrb.com/

**Reply by okuramasafumi** [reply]
https://middlemanapp.com/jp/
MiddlemanはKaigi on Railsで使っています！

**Comment by okuramasafumi** [comment]
【作ってる】
名前：Alba
種類：JSONシリアライザ
特徴：動作が速い、クラス定義なしでも使える
https://github.com/okuramasafumi/alba

【使ってる】
名前：benchmark-ips
種類：ベンチマークツール
特徴：正確、見やすい
https://github.com/evanphx/benchmark-ips

【気になってる】
名前：pretender
種類：Railsプラグイン（`current_user`の変更）
特徴：コード量が少ない（100行以下！）
https://github.com/ankane/pretender

**Comment by shoma07** [comment]
最近よく使う
- https://github.com/soutaro/steep (型チェックしてくれる)

検証中
- https://github.com/socketry/async (非同期まわり)
- https://github.com/jeremycole/mysql_binlog (binlogパーサー)

**Comment by danimal141** [comment]
stackprof
 https://github.com/tmm1/stackprof

**Comment by eiei19** [comment]
自分のプロダクトではこれ使います
役所とかのサーバーだとメール届かなくてつむことが
https://github.com/mikker/passwordless

**Comment by IkumaTadokoro** [comment]
昔よくお世話になったし、これのせいで苦しめられたこともあったのも含めて思い出があって好きです
https://github.com/r7kamura/view_source_map

**Comment by ryosk7** [comment]
好きなgem
factory_bot_rails
https://github.com/thoughtbot/factory_bot_rails

前から気になってるgem (おおくらさんいてびっくりしたw)
alba
https://github.com/okuramasafumi/alba

**Comment by genki0406** [comment]
メンテされてないけど昔からお世話になっているgem
https://github.com/valcker/similar_text-ruby

**Comment by sanfrecce-osaka** [comment]
(すみません、途中から参加していました 🙇 )
ruby-next
https://github.com/ruby-next/ruby-next
